### PR TITLE
Update ObsoleteAttribute remarks to account for new properties

### PR DESCRIPTION
**Pull request by gewarren**
_(Hide whitespace changes in diff.)_

### CODE REVIEW

File: xml/System/ObsoleteAttribute.xml [hunk 1]
```diff
@@ -54,43 +54,41 @@
...
+
+<xref:System.ObsoleteAttribute> is applicable to all program elements except assemblies, modules, parameters, and return values. Marking an element as obsolete informs users that the element may be removed in a future version of the product.
+
+The string assigned to the <xref:System.ObsoleteAttribute.Message%2A> property is emitted by the compiler when the attribute target is used in code. Ideally, the string should provide some workaround or programmatic alternative.
+
+Use the <xref:System.ObsoleteAttribute.IsError%2A> property to indicate to the compiler whether using the <xref:System.ObsoleteAttribute> attribute should cause it to emit an error (<xref:System.ObsoleteAttribute.IsError%2A> is `true`) or a warning (<xref:System.ObsoleteAttribute.IsError%2A> is `false`).
+
+For more information about using attributes, see [Attributes](/dotnet/standard/attributes/).
+
+## Windows 8.x Store apps
```
Comment (gewarren) [inline xml/System/ObsoleteAttribute.xml:69]: Do we still need this section?

Comment (jeffhandley) [reply xml/System/ObsoleteAttribute.xml:69]: > ## Windows 8.x Store apps
> Do we still need this section?

I'm not sure.  @terrajobst, what do you think?

File: xml/System/ObsoleteAttribute.xml [hunk 2]
```diff
@@ -247,16 +249,18 @@
...
+
+|Property|Value|
+|--------------|-----------|
+|<xref:System.ObsoleteAttribute.IsError%2A>|The `error` value.|
+|<xref:System.ObsoleteAttribute.Message%2A>|The `message` value.|
+|<xref:System.ObsoleteAttribute.DiagnosticId|`null`.|
+|<xref:System.ObsoleteAttribute.UrlFormat|`null`.|
+
  ]]></format>
         </remarks>
```
Comment (jeffhandley) [inline xml/System/ObsoleteAttribute.xml:265]: _This comment really applies down on line 292, but I can't add the comment there._

The `DiagnosticId` property's docs still have:

```xml
<remarks>To be added.</remarks>
```

I propose we update that to:

```xml
<remarks>The diagnostic ID is shown in build output for warnings and errors, and this is the unique ID that would be used to suppress the warnings or errors if needed.</remarks>
```

**Comment by gewarren** [comment]
[Preview URL](https://review.docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute.diagnosticid?view=net-5.0&branch=pr-en-us-4984).

**Review by jeffhandley — APPROVED** [review]

**Pull request merged**